### DISCUSSION TITLE
[BUG] GitHub.com profile URLs going crazy

### DISCUSSION THREAD

**Discussion by gonzalo-cordova-pou** [community/community#37535] [Other Feature Feedback, Questions, & Ideas]
When in my profile I go to "followers" or "following" and I click other user, I am redirected to the other user URL but I still see my profile pic, username, description, details... (left panel) but the URL, "Overview" and repos are the ones of the other user.

See it yourself:

https://user-images.githubusercontent.com/71346949/198346545-726688fb-87aa-430f-bc8a-1af0d01ae4ee.mp4

**Comment by radar-blip** [comment]
Fixed.

**Comment by jfullstackdev** [comment]
yup that's seems to be fixed by now I tested it a while ago too
